electrumsv/wallet_database/migrations/migration_0029_reference_server.py

```diff
@@ -71,11 +71,13 @@ def execute(conn: sqlite3.Connection, password_token: PasswordTokenProtocol,
     # We have persisted the next identifier for the `Accounts` table in the database.
     cursor = conn.execute("SELECT key, value FROM WalletData "
         "WHERE key='next_account_id' OR key='next_masterkey_id'")
-    wallet_data = { k: int(v) for (k, v) in cast(List[Tuple[str, str]], cursor.fetchall()) }
+    wallet_data: dict[str, Any] = { k: int(v) for (k, v) in cast(List[Tuple[str, str]],
```

Comment:
Why is this a dict of str:Any that sets all int values?

Reply:
because `wallet_data["main_server"] = ""` is setting the value to a string